docs/source/deployment/kubernetes/corda-kubernetes-deployment-option.rst

```diff
@@ -0,0 +1,175 @@
+This Node CA is then capable of generating the necessary TLS certificate and signing keys for use in transactions on the network.
+
+This process is generally initiated by executing ``java -jar corda.jar initial-registration``.
+The process will always need access to the Corda Network root truststore. This is usually assigned to the above command with additional parameters ``--network-root-truststore-password $TRUSTSTORE_PASSWORD --network-root-truststore ./workspace/networkRootTrustStore.jks``.
+The ``networkRootTrustStore.jks`` file should be placed in folder ``helm/files/network``.
+Once initiated the Corda Node will start the CSR request and wait indefinitely until the CSR request returns or is cancelled.
+If the CSR returns successfully, next the Node will generate the certificates in the folder ``certificates``.
+The generated files from this folder should then be copied to the following folder: ``helm/files/certificates/node``.
+
+CorDapps
+~~~~~~~~
+
+Any CorDapps that you would like to install on the Corda Node during the deployment should be placed in folder: ``helm/files/cordapps``.
+All files in this folder can then be automatically found in the Corda Node after executing ``pre-install.sh`` in folder ``helm/output/corda/templates``.
+
+Deployment
+~~~~~~~~~~
+
+At this point the one-time setup should be completed and we can move on to the actual deployment.
+We'll start with executing Helm to compile the template into a set of output files that we can directly apply into the Kubernetes cluster to create resources.
+
+Helm settings / configuration
+-----------------------------
+
+Helm works by taking values in a special file called ``values.yaml`` and applying those values to the files in folders ``files`` and ``templates``.
+The ``values.yaml`` file is well documented on its own and should be filled in carefully and diligently. Do not make typos here, double-check every value.
+
+Helm compile
+------------
+
+Now we are ready to compile the Helm templates into Kubernetes scripts.
+Executing ``helm_compile.sh`` will do the following:
+
+* Goes through the ``templates`` folder, for each file replaces variables by actual values from ``values.yaml`` file.
+* Places the compiled templates into the ``output/templates`` folder.
+* For the special file ``ConfigMap.yml``, combines the files in folder ``files`` into their raw text/binary representation and embeds them into ``ConfigMap.yml``. This file acts as way to inject files into Kubernetes pods.
+* Lastly applies the generated Kubernetes files directly into the Kubernetes cluster using ``kubectl apply -f``.
+
+Removing all installed resources from Kubernetes
+------------------------------------------------
+
```

Comment:
Suggestion and sentence update:

In the event you want a new fresh deployment of the Kubernetes cluster (especially useful during first few executions while learning the system), run the bash script called delete-all.sh. This file will *delete* Kubernetes cluster resources in the following order: 

- PersistentVolumes
- PersistentVolumeClaims
- StatefulSets
- Deployments
- Services
- Pods
- Jobs

The script will then wait until all the resources have been deleted from the Kubernetes cluster by polling it every 5 seconds. Finally it will remove any PersistentVolumeClaims that have been released in the shutdown phase. Once this script has finished executing, it is *then* safe to execute helm_compile.sh (or kubectl apply -f directly) to perform a fresh *deployment*.